### ISSUE TITLE
Uncaught TypeError: Cannot read property 'markBufferRange' of undefined

### ISSUE THREAD

**Issue by tomitrescak** [AtomLinter/Linter#610]
[Enter steps to reproduce below:]

1. Not sure, just started to appear.

**Atom Version**: 0.211.0
**System**: Mac OS X 10.10.3
**Thrown From**: [linter](https://github.com/AtomLinter/Linter) package, v1.0.4

### Stack Trace

Uncaught TypeError: Cannot read property 'markBufferRange' of undefined

```
At /Users/tomi/.atom/packages/linter/lib/linter-views.coffee:152

TypeError: Cannot read property 'markBufferRange' of undefined
  at /Users/tomi/.atom/packages/linter/lib/linter-views.coffee:152:45
  at Set.forEach (native)
  at LinterViews._renderPanel (/Users/tomi/.atom/packages/linter/lib/linter-views.coffee:149:16)
  at LinterViews._changeTab (/Users/tomi/.atom/packages/linter/lib/linter-views.coffee:126:8)
  at linter-bottom-tab.<anonymous> (/Users/tomi/.atom/packages/linter/lib/linter-views.coffee:23:8)

```

### Commands

```
     -0:56.3.0 command-palette:toggle (atom-text-editor.editor.is-focused)
  2x -0:53.7.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -0:52.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:52.6.0 project-manager:toggle (atom-text-editor.editor)
     -0:52.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-monokai"
    ],
    "disabledPackages": [
      "better-handlebars",
      "preview-plus",
      "tree-view-git-status",
      "sublime-tabs",
      "tab-history",
      "atom-handlebars",
      "meteor-helper",
      "enhanced-tabs",
      "close-tags"
    ],
    "ignoredNames": [
      ".settings",
      ".idea",
      ".git",
      ".DS_Store",
      "node_modules",
      "js",
      ".tmp",
      "documentation",
      ".meteor",
      ".build*",
      ".npm"
    ],
    "excludeVcsIgnoredPaths": false
  },
  "linter": {
    "subtleLinterErrors": [
      "tslint"
    ]
  }
}
```

### Installed Packages

```coffee
# User
.bin, vundefined
Handlebars, v1.1.2
atom-html-preview, v0.1.8
atom-jasmine, v0.6.0
atom-monokai, v0.8.0
atom-typescript, v4.6.5
autoclose-html, v0.18.0
build, v0.38.0
docblockr, v0.7.2
linter, v1.0.4
linter-tslint, v0.2.2
project-manager, v1.15.10
recent-files-fuzzy-finder, v0.1.4
zentabs, v0.8.4

# Dev
No dev packages
```

**Comment by k2b6s9j** [comment]
Could you please disable `linter-tslint` and report if the error still occurs? If it does then we know it is the fault of `atom-typescript`, if it doesn't then we know it is the fault of `linter-tslint`.

**Comment by tomitrescak** [comment]
Yes, rolling back to version < 1.0 (0.12.*) has disabled this error.
I got this error mostly appearing when opening system preferences and JSON files.

**Comment by k2b6s9j** [comment]
By rolling back to < 1.0 you have confirmed that it is the fault of a provider implementing the new API, i.e. `atom-typescript`. Please open an issue in their repository with the stack trace you've posted above.

**Comment by steelbrain** [comment]
@k2b6s9j fix is up in #614
